CHANGES.md

```diff
@@ -1,3 +1,7 @@
+## 0.16.8
+
+* Fixed a bug that lead to data corruption when encoding with Average filter.
```

Comment:
nit: lead -> led

Reply:
Thanks. I swear, I'm not doing that on purpose to check if you're reading them :wink: